qemu/tests/block_copy.py

```diff
@@ -298,7 +297,10 @@ def action_when_start(self):
                 bg = utils.InterruptedThread(fun)
                 bg.start()
                 if bg.isAlive():
-                    self.processes.append(bg)
+                    try:
```

Comment:
For example:
1. system_reset
2. do live snapshot
3. verify alive

Current problem is that verify guest alive failed as reboot is not finished.